programs/mmm/src/instructions/admin/create_pool.rs

```diff
@@ -75,7 +78,10 @@ pub fn handler(ctx: Context<CreatePool>, args: CreatePoolArgs) -> Result<()> {
     pool.payment_mint = args.payment_mint;
     pool.allowlists = args.allowlists;
 
-    log_pool("post_create_pool", pool)?;
```

Comment:
Yes, that's correct. The idea behind this PR is to change the way logging happens to avoid truncated logs causing misses on indexing, but it requires updating the indexing parser to handle the new logging method. So the indexing changes should be implemented before this PR is merged. @madergaser knows what needs to happen on this.